dotnet/docker/dotnet_release_app.dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM mcr.microsoft.com/dotnet/sdk:8.0-bookworm-slim AS build_app
+
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends  openssh-client
+
+RUN set -x \
+    && git clone https://github.com/DataDog/dotnet-aas-samples.git \
+    && cd dotnet-aas-samples/Junkyard.Web \
+    && dotnet publish -c Release --framework net8.0 -o /app
+
+FROM mcr.microsoft.com/dotnet/sdk:8.0-bookworm-slim AS setup_apm
+
+ARG APM_COMMIT_SHA
+
+ARG ARTIFACT_NAME="datadog-dotnet-apm.tar.gz"
+
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends  xq
+
+# Retrieve and install .NET APM
+RUN set -x \
+    && curl -sS https://apmdotnetci.blob.core.windows.net/apm-dotnet-ci-artifacts-master/\?comp\=list\&prefix\=${APM_COMMIT_SHA} |  xq -x EnumerationResults/Blobs/Blob/Url | grep -E 'datadog-dotnet-apm-[0-9]+\.[0-9]+\.[0-9]+\.tar\.gz' | xargs curl -sSL -o ${ARTIFACT_NAME} \
+    && mkdir -p /opt/datadog \
+    && tar zxvf ${ARTIFACT_NAME} -C /opt/datadog
+
+# final stage/image
+FROM mcr.microsoft.com/dotnet/aspnet:8.0-bookworm-slim
+
+ARG APM_COMMIT_SHA
+
+# setup SSH for debugging/investigation (in Azure portal)
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends openssh-server
+
+RUN echo "root:Docker!" | chpasswd
+RUN ssh-keygen -A
+
+COPY sshd_config /etc/ssh/
```

Comment:
Taking your word for it that this is ok 😅

Reply:
Tested yes :)